### PR TITLE
 Update Badge column so you can update values

### PR DESCRIPTION
**Pull request by PurpleBabar**

### CODE REVIEW

File: src/components/tables/BadgeColumn.vue [hunk 1]
```diff
@@ -14,15 +14,18 @@
...
-    data () {
-      return {
-        classObject: {
-          'badge-warning': this.rowIndex % 6 === 0,
-          'badge-primary': this.rowIndex % 6 === 1,
-          'badge-danger': this.rowIndex % 6 === 2,
-          'badge-info': this.rowIndex % 6 === 3,
-          'badge-violet': this.rowIndex % 6 === 4,
-          'badge-dark-blue': this.rowIndex % 6 === 5
+    data: {
```
Comment (smartapant) [inline src/components/tables/BadgeColumn.vue]: Do we need classObject in data?

File: src/components/tables/BadgeColumn.vue [hunk 2]
```diff
@@ -14,15 +14,18 @@
...
+      classObject: {}
```
Comment (smartapant) [inline src/components/tables/BadgeColumn.vue]: Do we need classObject in data?

**Comment by PurpleBabar** [comment]
I modified it :)

**Comment by smartapant** [comment]
@PurpleBabar good, thanks

**Pull request merged**